docs/application/tizen-studio/common-tools/keyboard-shortcuts.md

```diff
@@ -191,7 +191,7 @@ The following tables introduce the keyboard shortcuts available for the various
 
 **Table: ECP shortcuts**
 
-| Command  | Ubuntu                                   | Windows®        | macOS           |                 |
+| Command  | Ubuntu                                   | Windows&reg;        | macOS           |                 |
```

Comment:
The 1st row of this table is shifted to left side by one cell.

Reply:
done

Reply:
done